lib/src/tasks/coverage/api.dart

```diff
@@ -317,8 +331,14 @@ class CoverageTask extends Task {
 
     Map mergedJson = JSON.decode(collections.first.readAsStringSync());
     for (int i = 1; i < collections.length; i++) {
-      Map coverageJson = JSON.decode(collections[i].readAsStringSync());
-      mergedJson['coverage'].addAll(coverageJson['coverage']);
+      if (!collections[i].existsSync()) continue;
+      String coverage = collections[i].readAsStringSync();
+      if (coverage.isEmpty) {
+//        mergedJson['coverage'].addAll({});
```

Comment:
If this isn't needed, get rid of this if block altogether and just use `if (coverage.isNotEmpty) {`